client/src/placement-context.tsx

```diff
@@ -89,14 +90,17 @@ export function PlacementProvider(props: { children: React.ReactNode }) {
       }
 
       try {
-        const placementResponse: PlacementContextData = await response.json();
+        const {
+          plusAvailable = true, // Fall back to true for seamless migration.
+          ...placementResponse
+        }: PlacementContextData = await response.json();
```

Comment:
I would have preferred this 1 line change:

```suggestion
        const placementResponse: PlacementContextData = await response.json();
        placementResponse.plusAvailable ??= true; // Fall back for seamless migration.
```